lib/maze/options/option_validator.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require_relative '../option'
+require_relative '../capabilities/devices'
+
+module Maze
+  module Options
+    # Validates command line options
+    class OptionValidator
+      # Validates all provided options
+      # @param options [Hash] Parsed command line options
+      def validate(options)
+        errors = []
+
+        # Common options
+        farm = options[Option::FARM]
+        if farm && !%w[bs local].include?(farm)
+          errors << "--#{Maze::Option::FARM} must be either 'bs' or 'local' if provided"
+        end
+
+        begin
+          JSON.parse(options[Option::CAPABILITIES])
+        rescue JSON::ParserError
+          errors << "--#{Maze::Option::CAPABILITIES} must be valid JSON (given #{options[Option::CAPABILITIES]})"
```

Comment:
Some places in this file are using `Option::<name>` but others do `Maze::Option::<name>`

Reply:
All changed to the shorter form